sustainability/models/carbon_factor.py

```diff
@@ -492,7 +492,7 @@ def action_see_child_ids(self):
         return self._generate_action(
             title=_("Child factors for"),
             model="carbon.factor",
-            ids=self._get_distribution_lines_res_ids("carbon.factor"),
+            ids=list(self.child_ids.mapped("id")),
```

Comment:
```suggestion
            ids=self.child_ids.ids,
```

More "Odoo" way to do it